src/stable_browser.ts

```
@@ -1901,6 +1901,10 @@ class StableBrowser {
         case "disabled":
           val = String(await state.element.isDisabled());
           break;
+        case "readonly":
+          const isEditable = await state.element.isEditable();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)